python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -3,5 +3,8 @@
 def home(request):
     return render_to_response('home.html')
 
+def map(request):
```

Comment:
A bit late to the party, but +1 for function-based views; I doubt this is going to be the type of app where class-based views would really shine.